From e318842f88df0dca6cae58b752f47606515a2e43 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Tue, 25 Sep 2018 13:51:22 -0700
Subject: [PATCH 03/13] DO-NOT-UPSTREAM: Disable -Wduplicate-decl-specifier for
 Clang

Clang currently warns on 'const typeof'. This has been fixed upstream
and this patch will be dropped when I get a chance to rebuild my
toolchain.

Link: https://github.com/ClangBuiltLinux/linux/issues/52
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 3ab7faacf2c9..b2af0f883add 100644
--- a/Makefile
+++ b/Makefile
@@ -709,6 +709,7 @@ KBUILD_CFLAGS += $(call cc-disable-warning, gnu)
 KBUILD_CFLAGS += $(call cc-disable-warning, address-of-packed-member)
 # Quiet clang warning: comparison of unsigned expression < 0 is always false
 KBUILD_CFLAGS += $(call cc-disable-warning, tautological-compare)
+KBUILD_CFLAGS += $(call cc-disable-warning, duplicate-decl-specifier)
 # CLANG uses a _MergedGlobals as optimization, but this breaks modpost, as the
 # source of a reference will be _MergedGlobals and not on of the whitelisted names.
 # See modpost pattern 2
-- 
2.19.0

